torch_geometric/explain/algorithm/dummyexplainer.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+import torch
+
+from torch_geometric.data import Data
+from torch_geometric.explain.base import ExplainerAlgorithm
+from torch_geometric.explain.explanations import Explanation
+
+
+class RandomExplainer(ExplainerAlgorithm):
+    """Dummy explainer."""
+    def forward(self, x: torch.Tensor, edge_index: torch.Tensor,
+                edge_attr: Optional[torch.Tensor] = None,
+                **kwargs) -> Explanation:
+        """Returns a random explanation based on the shape of the inputs.
+
+        Args:
+            x (torch.Tensor): node features.
+            edge_index (torch.Tensor): edge indices.
+            edge_attr (torch.Tensor, optional): edge attributes.
+                Defaults to None.
+
+        Returns:
+            Explanation: random explanation based on the shape of the inputs.
+        """
+        node_features_mask = torch.rand(x.shape)
+        node_mask = torch.rand(x.shape[0])
+        edge_mask = torch.rand(edge_index.shape[1])
+        masks = (node_mask, edge_mask, node_features_mask)
+        mask_keys = ("node_mask", "edge_mask", "node_features_mask")
```

Comment:
Why not save in a dictionary?
```python
mask_dict = {
    'node_mask': node_mask,
    ...
}
```
This way, we don't need to track both `masks` and `mask_keys`.